api/forgeops_cli_types.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import "errors"
+
+// ResultStatus result of a command
+type ResultStatus string
+
+var (
+	// ResultStatusSuccess command completed without errors
+	ResultStatusSuccess ResultStatus = "Success"
+	// ResultStatusFailure command completed with errors
+	ResultStatusFailure ResultStatus = "Failed"
+)
+
+// ResultMessage is amessage that contains a result from the command. The struture is a key value, where value is a string
+type ResultMessage map[string]string
+
+// NewResultFromKeyPair build a result message from key value pairs
+func NewResultFromKeyPair(kp []string) (*ForgeOpsResult, error) {
```

Comment:
why not use a map?

Reply:
See the other comment about this. I'm mostly trying to match the logging api, plus not allocate a map. Now I'm using a variadic of strings